discord_slash/model.py

```diff
@@ -358,3 +379,64 @@ async def wrap():
                     await self._http.delete(self.__interaction_token, self.id)
 
             self._state.loop.create_task(wrap())
+
+
+class PermissionData:
+    """
+    Single slash permission data.
+
+    :ivar id: User or role id, based on following type specfic.
+    :ivar type: The ``SlashCommandPermissionsType`` type of this permission.
+    :ivar permission: State of permission. ``True`` to allow, ``False`` to disallow.
+    """
+    def __init__(self, id, type, permission, **kwargs):
+        self.id = id
+        self.type = type
+        self.permission = permission
+
+    def __eq__(self, other):
+        if isinstance(other, PermissionData):
+            return (
+                self.id == other.id
+                and self.type == other.id
+                and self.permission == other.permission
+            )
+        else:
+            return False
+
+
+class GuildPermissionsData:
+    """
+    Slash permissions data for a guild.
+
+    :ivar id: Command id, provided by discord.
+    :ivar permissions: List of permissions dict.
```

Comment:
https://discord.com/developers/docs/interactions/slash-commands#guildapplicationcommandpermissions

```suggestion
    Slash permissions data for a command in a guild.

    :ivar id: Command id, provided by discord.
    :ivar guild_id: Guild id that the permissions are in. 
    :ivar permissions: List of permissions dict.
```

Reply:
Code also needs to be updated for this

Reply:
@benwoo1110 I unresolved this before, see above ^^

Reply:
oh, I thought it was just a docs typo

Reply:
89d38c0